rs/ethereum/ledger-suite-orchestrator/src/state/mod.rs

```diff
@@ -698,3 +793,89 @@ pub fn init_state(state: State) {
             .expect("failed to initialize state in stable cell")
     });
 }
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct ManageOtherCanisters {
```

Comment:
This does not seem too different than the one defined in `candid/mod.rs` of the same name. Is there any chance to use only one definition?

Reply:
The main differences are that this struct in `state/mod.rs` uses the domain structs (such as `WasmHash`). So basically everything in the domain should use `state::ManageOtherCanisters`, because it was validated, whereas `candid::ManageOtherCanisters` contains unvalidated user input and should be used in very limited places (basically just deserializing user input). It's also usually a good idea to decouple the types that are externally visible (`candid::ManageOtherCanisters`) and have a character API from the internal ones (`state::ManageOtherCanisters`). WDYT?

Reply:
Make sense. Thanks!